src/components/Tokens/TokenTable/TokenTable.tsx

```diff
@@ -75,7 +75,7 @@ export default function TokenTable() {
 
   // TODO: consider moving prefetched call into app.tsx and passing it here, use a preloaded call & updated on interval every 60s
   const chainName = validateUrlChainParam(useParams<{ chainName?: string }>().chainName)
-  const { error, loading, tokens, hasMore, loadMoreTokens, maxFetchable } = useTopTokens(chainName)
+  const { error, loading, tokens, hasMore, loadMoreTokens, loadingRowsCount } = useTopTokens(chainName)
```

Comment:
nit: `loadingRowCount` (not `Rows`) to match the prop that it is being passed in to.